### PR TITLE
GITLAB-779 Publish docker images when building branches

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: workflow-job [hunk 1]
```diff
@@ -519,24 +530,25 @@ flows["${sha1}"] = {
 			//Publishing to Docker Hub
 			def dUser = dockerUser();
 			def dPassword = dockerPassword();
+                        def tag = tag();
```
Comment (rouazana) [inline workflow-job]: indent

**Review by rouazana — COMMENTED** [review]
nice, but are branches and tags automatically built when pushing them, or do we need to specifically activate some build process for them?

File: workflow-job (hunk 1 above)
Comment (aduprat) [reply workflow-job]: That was a mistake, we don't need that.

**Comment by aduprat** [comment]
https://github.com/linagora/james-jenkins/pull/128

**Pull request closed without merging**